aztec/src/main/kotlin/org/wordpress/aztec/formatting/InlineFormatter.kt

```diff
@@ -238,13 +262,6 @@ class InlineFormatter(editor: AztecText, val codeStyle: CodeStyle) : AztecFormat
         joinStyleSpans(start, end)
     }
 
-    fun removeInlineCssStyle(start: Int = selectionStart, end: Int = selectionEnd) {
```

Comment:
I am a little suspicious that removing this code might have side effects, though I'm not sure what they are yet. Also this does not seem to fix all cases of https://github.com/wordpress-mobile/AztecEditor-Android/issues/914. For example, if you highlight the first red italic text in the sample text and turn off italic, and then open html you will see the color attribute is gone, and returning from html the color will be gone in the display text. 

I did see that background color no longer disappears when you remove a text format so that is an improvement. Perhaps we leave this code as it was since it does not affect background, and it does not seem to completely fix https://github.com/wordpress-mobile/AztecEditor-Android/issues/914 either.

Reply:
This code is weird, it just remove everything without any rules, not sure if it will break something, maybe it's a work around to something? it really doesn't make any sense to me